quickwit-indexing/Cargo.toml

```diff
@@ -49,11 +49,11 @@ rusoto_kinesis = { version = "0.48", default-features = false, features = [
 serde = "1"
 serde_json = "1"
 serde_yaml = "0.8"
-tantivy = { git = "https://github.com/quickwit-oss/tantivy/", rev = "2406d92", default-features = false, features = [
-    "mmap",
-    "lz4-compression",
-    "zstd-compression",
-    "quickwit",
+tantivy = { git = "https://github.com/quickwit-oss/tantivy/", rev = "d24f31f", default-features = false, features = [
+  "mmap",
+  "lz4-compression",
+  "zstd-compression",
+  "quickwit"
```

Comment:
lol, just review a PR from Paul where he added the trailing comma. This makes me feel better about myself, I felt I was the only one OCD on the team.